src/server/handlers/webhook.js

```diff
@@ -83,5 +115,23 @@ export const notify = async (req, res) => {
       logger.error(`Failed to request builds of ${repositoryUrl}: ${error}.`);
       return res.status(500).send();
     }
+  } else if (req.headers['x-launchpad-event-type'] === 'snap:build:0.1') {
```

Comment:
Maybe it would be clearer to have separate function to handle github and launchpad webhooks.

It's not terrible now, as there are only 2 of them. But if we get to the point where this `if else` grows even more it will be quite hard to comprehend.